### PR TITLE
Release 202212.0

### PR DESCRIPTION
**Pull request by dmiseev**
### Overview

Release 202212.0

### CODE REVIEW

**Review by FRITS-Kh — APPROVED** [review]

File: deploy.aop-b2b-testing.yml [hunk 1]
```diff
@@ -0,0 +1,274 @@
+version: "0.1"
```
Comment (DmitryLymarenko) [inline deploy.aop-b2b-testing.yml:1]: reminder to remove this one as soon as smoke testing is completed

File: deploy.aop-demoshop-b2b.yml [hunk 2]
```diff
@@ -0,0 +1,262 @@
+version: "0.1"
```
Comment (DmitryLymarenko) [inline deploy.aop-demoshop-b2b.yml:1]: reminder to remove this one as soon as smoke testing is completed

File: deploy.spryker-b2b-eu.yml [hunk 3]
```diff
@@ -0,0 +1,216 @@
+version: "0.1"
```
Comment (DmitryLymarenko) [inline deploy.spryker-b2b-eu.yml:1]: reminder to remove this one as soon as smoke testing is completed

File: deploy.spryker-b2b-intt.yml [hunk 4]
```diff
@@ -0,0 +1,165 @@
+version: "0.1"
```
Comment (DmitryLymarenko) [inline deploy.spryker-b2b-intt.yml:1]: reminder to remove this one as soon as smoke testing is completed

File: deploy.spryker-b2b-production.yml [hunk 5]
```diff
@@ -0,0 +1,218 @@
+version: "0.1"
```
Comment (DmitryLymarenko) [inline deploy.spryker-b2b-production.yml:1]: reminder to remove this one as soon as smoke testing is completed

File: deploy.spryker-b2b-staging.yml [hunk 6]
```diff
@@ -0,0 +1,221 @@
+version: "0.1"
```
Comment (DmitryLymarenko) [inline deploy.spryker-b2b-staging.yml:1]: depending on TW needs we will remove this one after smoke testing, or keep it as it is to refer in the documentation

File: deploy.spryker-b2b-us.yml [hunk 7]
```diff
@@ -0,0 +1,192 @@
+version: "0.1"
```
Comment (DmitryLymarenko) [inline deploy.spryker-b2b-us.yml:1]: reminder to remove this one as soon as smoke testing is completed

File: deploy.spryker-b2binttus.yml [hunk 8]
```diff
@@ -0,0 +1,146 @@
+version: "0.1"
```
Comment (DmitryLymarenko) [inline deploy.spryker-b2binttus.yml:1]: reminder to remove this one as soon as smoke testing is completed

File: deploy.spryker-b2bsec.yml [hunk 9]
```diff
@@ -0,0 +1,167 @@
+version: "0.1"
```
Comment (DmitryLymarenko) [inline deploy.spryker-b2bsec.yml:1]: reminder to remove this one as soon as smoke testing is completed

**Review by DmitryLymarenko — COMMENTED** [review]
Nothing prevents us from starting smoke testing
@dmiseev please prepare PR with the removal of mentioned deploy files and use `release-202212.0` as a target, so we won't waste time later

**Review by DmitryLymarenko — APPROVED** [review]

**Pull request merged**